<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPage</class>
 <widget class="QWidget" name="MenuPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="MainFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QFrame" name="leftToolFrame">
        <property name="minimumSize">
         <size>
          <width>210</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>*各种快捷按钮放这里（可以自定义</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>所以需要工厂函数注册或者json存储</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="centerMainFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>210</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="menuUpMainFrame">
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="menuTimeFrame">
                 <property name="frameShape">
                  <enum>QFrame::Shape::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Shadow::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="timeChooserLabel">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>50</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Time Span 选择者</string>
                    </property>
                    <property name="indent">
                     <number>100</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="menuFastFrame">
                    <property name="frameShape">
                     <enum>QFrame::Shape::NoFrame</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Shadow::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="fastSwitchLabel">
                       <property name="text">
                        <string>fast switch</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="timeChooser"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="menuSlowTimeFrame">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>80</height>
                     </size>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::Shape::NoFrame</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Shadow::Raised</enum>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item row="0" column="0">
                      <widget class="QLabel" name="yearLabel">
                       <property name="text">
                        <string>Year</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QComboBox" name="yearCombo"/>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="monthLabel">
                       <property name="text">
                        <string>Month</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QComboBox" name="monthCombobox"/>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="dayLabel">
                       <property name="text">
                        <string>Day</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="1">
                      <widget class="QComboBox" name="dayCombobox"/>
                     </item>
                     <item row="3" column="1">
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>以后看一下怎么解决这里太宽的问题,and设计逻辑</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="exploitFrame">
                 <property name="minimumSize">
                  <size>
                   <width>330</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::Shape::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Shadow::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QLabel" name="exploitationLabel">
                    <property name="text">
                     <string>高价值时间利用率</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="bigNumLabel">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>120</pointsize>
                      <italic>true</italic>
                      <bold>true</bold>
                      <underline>false</underline>
                      <strikeout>false</strikeout>
                     </font>
                    </property>
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;text!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignmentFlag::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="smallPercentLabel">
                    <property name="text">
                     <string>%</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="menuDownMainFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="fourRealmFrame">
              <property name="minimumSize">
               <size>
                <width>330</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Plain</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="extremeDataText"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="pageSwitchFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QToolButton" name="menuButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>AppleGothic</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="menuButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>AppleGothic</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Capture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="demoButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>AppleGothic</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Analysis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="InputButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>AppleGothic</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Strategy and Settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
